rust/main/chains/hyperlane-ethereum/src/contracts/validator_announce.rs

```diff
@@ -159,7 +163,11 @@ where
             return None;
         };
 
-        let Ok(balance) = self.provider.get_balance(eth_h160, None).await else {
+        let chain_signer = chain_signer
+            .and_then(|s| ethers::types::H160::from_str(&s).ok())
```

Comment:
Not sure if we should fallback to validator key or force to always use chain signer by making chain_signer not an `Option`

Reply:
I guess might as well make it not optional?